tools/interactive/interactivetool_ml_jupyter_notebook.xml

```diff
@@ -88,6 +97,7 @@ except FileNotFoundError:
         export HOME=/home/\$NB_USER/ &&
         export PATH=/home/\$NB_USER/.local/bin:\$PATH &&
         cp '${galaxy_input_startup_script}' /home/\$NB_USER/.ipython/profile_default/startup/02-load.py &&
+        jupyter labextension disable "@jupyterlab/apputils-extension:announcements" &&
```

Comment:
Can you maybe add a comment why we do this?
Can we remove it from the container instead?

Reply:
I added a comment.

I think those popus might be useful for someone that uses the docker image locally as opposed to a disposable instance in galaxy. So maybe we should not disable it from the docker recipe?